Description: Updates to configure.ac for package build.
 In the configure.ac file, change the OWNER setting from "mbse" to "mbsebbs",
 and change the AC_PREFIX_DEFAULT from "/opt/mbse" to just "".
 .
 This is for the Debian package build.
Author: Robert James Clay <jame@rocasa.us>
Last-Updated: 2016-09-08

--- a/configure.ac
+++ b/configure.ac
@@ -17,7 +17,7 @@
 COPYRIGHT="Copyright (C) 1997-2016 Michiel Broek, All Rights Reserved"
 SHORTRIGHT="Copyright (C) 1997-2016 M. Broek"
 GROUP="bbs"
-OWNER="mbse"
+OWNER="mbsebbs"
 ROWNER="`id -un root`"
 RGROUP="`id -gn root`"
 AC_SUBST(PACKAGE)
@@ -29,7 +29,7 @@
 AC_SUBST(OWNER)
 AC_SUBST(ROWNER)
 AC_SUBST(RGROUP)
-AC_PREFIX_DEFAULT(/opt/mbse)
+AC_PREFIX_DEFAULT(/usr/lib/mbsebbs)
 AC_DEFINE_UNQUOTED(VERSION, "$VERSION")
 AC_DEFINE_UNQUOTED(VERSION_MAJOR, $MAJOR)
 AC_DEFINE_UNQUOTED(VERSION_MINOR, $MINOR)
